src/main/java/core/mate/academy/service/TruckProducerImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducerImpl implements MachineProducer<Truck> {
+
+    @Override
+    public List<Truck> get() {

```

Comment:
Consider creating a local variable inside the `get` method to hold the list of trucks before returning it. This can increase the readability of the code.